### PR TITLE
[stable/grafana] deprecate chart

### PR DESCRIPTION
**Pull request by zanhsieh**
This chart was moved to a new location: https://github.com/grafana/helm-charts 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by torstenwalter** [comment]
@zanhsieh You need to add `deprecated: true` Like https://github.com/helm/charts/blob/master/stable/jenkins/Chart.yaml then the linting error should be gone.

**Comment by scottrigby** [comment]
@zanhsieh May I also make a few suggestions about the new repo?
- bring over git history from stable (here are steps to help: https://github.com/scottrigby/prometheus-helm-charts/issues/3)
- add CI/CD using Helm GitHub Actions (working demo repo here, with steps in the README https://github.com/helm/charts-repo-actions-demo)
- Invite the existing chart maintainers to help with that repo https://github.com/grafana/helm2-grafana/blob/master/OWNERS
- list the new repo in Artifact Hub https://artifacthub.io/ (the successor of Helm Hub. See https://github.com/helm/hub/issues/439. But also if you wish you may list at Helm Hub in the meantime)

File: stable/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,19 +1,13 @@
+# This chart is deprecated and moved to https://github.com/grafana/helm2-grafana
 apiVersion: v1
 name: grafana
```
Comment (davidkarlsen) [inline stable/grafana/Chart.yaml:3]:
```suggestion
name: grafana
deprecated: true
```

File: stable/grafana/templates/NOTES.txt [hunk 2]
```diff
@@ -1,3 +1,8 @@
+*******************
+****DEPRECATED*****
+*******************
+* The Jenkins chart is deprecated. Future development has been moved to https://github.com/grafana/helm2-grafana
```
Comment (davidkarlsen) [inline stable/grafana/templates/NOTES.txt]:
```suggestion
* The chart is deprecated. Future development has been moved to https://github.com/grafana/helm2-grafana
```

**Comment by mrueg** [comment]
@zanhsieh out of curiosity, why is the repo called helm2? Is there one for helm3?

**Comment by torstenwalter** [comment]
You could simplify your life if you would call the repository helm-charts and put both charts in a subfolder named `charts`

**Comment by scottrigby** [comment]
Hm, @zanhsieh re https://github.com/helm/charts/pull/23662#issuecomment-685085598 I also don't believe this should have been merged yet. As far as I can tell, the chart at https://github.com/grafana/helm2-grafana is not installable for end users (or if it is, the Readme still instructs users to install `stable/grafana`).

**Comment by torstenwalter** [comment]
@zanhsieh I could help with the migration. If you want I can prepare a repo including the history, add you as admin and you could transfer it to https://github.com/grafana/.
Could also setup github actions for it similar https://github.com/jenkinsci/helm-charts or https://github.com/scottrigby/prometheus-helm-charts.

Just let me know if you want that help.

**Comment by unguiculus** [comment]
I very much second what @scottrigby and @torstenwalter say. Bring it over including the history. Also, the chart should must in a subfolder named after the chart.

**Comment by unguiculus** [comment]
Also, it's probably not a good idea to have separate charts for Helm 2 and 3. That would be two codebases for the same thing with the risk that things diverge. I'd stick with Helm 2 for now which is also installable with Helm 3 until Helm 2 reaches EOL which is pretty soon anyways. You can then update the chart for Helm 3 and drop Helm 2 compatibility.

**Comment by zanhsieh** [comment]
@torstenwalter 
I need help with the migration as well as other stuffs, e.g. github actions, branching. Had invited you in helm2-grafana chart. 

@unguiculus 
Sure.

**Comment by paulczar** [comment]
I see there are a bunch of charts in the loki repo - https://github.com/grafana/loki/tree/master/production/helm

maybe it would make sense for the grafana folks to create https://github.com/grafana/helm-charts and consolidate them all into one place ?

**Comment by torstenwalter** [comment]
Accepted the invite. Which charts should be migrated there? Just grafana or also loki? If you list them here then we can migrate them in one go (will be easier).

**Comment by zanhsieh** [comment]
@torstenwalter 
Just grafana chart. I don't have control on loki. 

@paulczar 
That part is hard for me since have to talk to Daniel Lee from Grafana. I don't work for Grafana; all my previously contact is thru email.

**Comment by scottrigby** [comment]
@torstenwalter fun idea. you could also bring in loki, and start up the convo with the loki maintainers to see if they'd like to have all the ci/cd and maintenance in once place (if there's a concern about permissions, mention GitHub CODEOWNERS as the solution, as we did for prometheus-community). If they decide to keep it there, the work is done. If they prefer not to have all the grafana charts in the same repo, you would just need to filter it out, and everything else will stay the same. WDYT?

**Comment by torstenwalter** [comment]
Started the discussion in https://github.com/grafana/loki/issues/2593

**Comment by torstenwalter** [comment]
Short update: chart is now migrated including history to https://github.com/grafana/helm-charts and GitHub workflows are set up. Deprecation notice will be updated with #23686.

**Pull request merged**